administrator/language/de-DE/plg_workflow_featuring.ini

```diff
@@ -4,9 +4,9 @@
 ; License GNU General Public License version 2 or later; see LICENSE.txt
 ; Note : All ini files need to be saved as UTF-8
 
-PLG_WORKFLOW_FEATURING="Workflow - Featuring"
-PLG_WORKFLOW_FEATURING_CHANGE_STATE_NOT_ALLOWED="You're not allowed to change the featured state of this item. Please use a workflow transition."
-PLG_WORKFLOW_FEATURING_FEATURED="Featured: %s"
-PLG_WORKFLOW_FEATURING_TRANSITION_ACTIONS_FEATURING_DESC="Define the featured state an item should be set, when executing this transition"
-PLG_WORKFLOW_FEATURING_TRANSITION_ACTIONS_FEATURING_LABEL="Featuring state"
-PLG_WORKFLOW_FEATURING_XML_DESCRIPTION="Add featuring actions to the workflow transitions for your items"
+PLG_WORKFLOW_FEATURING="Workflow – Haupt-Eigenschaft"
+PLG_WORKFLOW_FEATURING_CHANGE_STATE_NOT_ALLOWED="Es ist nicht zulässig, den Haupt-Status dieses Eintrags zu ändern. Bitte einen Workflow-Übergang verwenden."
+PLG_WORKFLOW_FEATURING_FEATURED="Haupteintrag: %s"
+PLG_WORKFLOW_FEATURING_TRANSITION_ACTIONS_FEATURING_DESC="Den Status des Eintrags definieren, der bei der Ausführung dieses Übergangs gesetzt werden soll"
+PLG_WORKFLOW_FEATURING_TRANSITION_ACTIONS_FEATURING_LABEL="Haupt-Status"
+PLG_WORKFLOW_FEATURING_XML_DESCRIPTION="Den Workflow-Übergängen ihrer Einträge bestimmte Aktionen hinzufügen"
```

Comment:
```suggestion
PLG_WORKFLOW_FEATURING_XML_DESCRIPTION="Fügt den Workflow-Übergängen der Einträge besondere Optionen hinzu"
```

Reply:
Im Souce-File ist hier von "Add featuring actions..." die Rede
Anders in der .sys.ini – dot wird von "options" gesprochen

Ist das ein Fehler in der en-GB?

Reply:
> Ist das ein Fehler in der en-GB?

Ja. Ist nur die Frage was richtig ist. Kriegen wir dann aber ja in einem neuen Issue mit. Aber Aktionen ist sicherlich dann das richtige.

Reply:
:confused: 
wie jetzt? Aktionen oder Optionen?